git.hooks code/commit-msg

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+popen('chmod +x commit_msg', shell=True, stdout=PIPE)
```

Comment:
To get this code line executed `commit_msg` should have +x flag, but if it will have this flag, then you don't need this do it by yourself.

Reply:
this string is absent in master branch, I deleted this file from current branch to save the file in master after merge